src/patternfly/components/Button/button.scss

```diff
@@ -122,10 +122,10 @@
   --#{$button}--m-plain--m-clicked--BackgroundColor: var(--pf-t--global--background--color--action--plain--clicked);
   --#{$button}--m-plain--disabled--Color: var(--pf-t--global--icon--color--disabled);
   --#{$button}--m-plain--disabled--BackgroundColor: transparent;
-  --#{$button}--m-plain--m-small--PaddingTop: var(--pf-t--global--spacer--xs);
-  --#{$button}--m-plain--m-small--PaddingRight: var(--pf-t--global--spacer--xs);
-  --#{$button}--m-plain--m-small--PaddingBottom: var(--pf-t--global--spacer--xs);
-  --#{$button}--m-plain--m-small--PaddingLeft: var(--pf-t--global--spacer--xs);
+  --#{$button}--m-plain--m-small--PaddingTop: calc(var(--pf-t--global--spacer--button--vertical--plain) / 2);
```

Comment:
@lboehling There are several instances where I'm taking an existing semantic spacer and dividing/multiplying them to get a correct final value. Would we want to add some more semantic spacers for these scenarios?

Reply:
i think you could use --pf-t--global--spacer--button--vertical--compact here instead of dividing/multiplying? that should give the correct value?

Reply:
Would other instances of multiplying/dividing be okay for now? For example line 220 of the button.scss file